core/pom.xml

```diff
@@ -165,25 +173,13 @@
     <dependency>
       <groupId>org.jenkins-ci</groupId>
       <artifactId>test-annotations</artifactId>
-      <version>1.4</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.hamcrest</groupId>
       <artifactId>hamcrest</artifactId>
-      <version>2.2</version>
```

Comment:
Defining the version centrally in `stapler-parent`'s `<dependencyManagement>` section to avoid duplication.